internal/template_test.go

```diff
@@ -22,23 +22,23 @@ var directories = []struct {
 	name             string
 }{
 	{
-		workingDirectory: "./test",
+		workingDirectory: "test",
```

Comment:
The `workingDirectory` is returned in the tests from the stubbed `createDirectory` function. The real function does not add a `./` and so removing this in order to be consistent